build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildFailureException-BuildTarget.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
To avoid a dependency on Microsoft.DotNet.Cli.Utils, and avoid having another copy (we already have 3) of command, argumentescaper, etc... 

These are common utility classes. I don't see why the source include is wrong as it's preventing worse things.